.github/workflows/release-checklist.yml

```diff
@@ -0,0 +1,112 @@
+name: '[ci] release-checklist'
+
+on:
+  push:
+    tags: ["**"]
+
```

Comment:
Wouldn't we want this to run only if tests are passing?

Reply:
if tests ain't passing, we shouldn't be tagging releases